sdk/core/core-util/test/public/reshape.spec.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { describe, it, assert, expect } from "vitest";
+import { reshape } from "../../src/reshape";
+
+describe("reshape", function () {
```

Comment:
Is it possible that `input` could be an array?

Reply:
Seems to work:

```ts
it("can handle arrays as the initial input", function () {
    const input = [1, 2, 3];
    const actual = reshape(input, "[]", String);
    const expected = ["1", "2", "3"];
    assert.deepEqual(actual, expected);
  });
```